package.json

```diff
@@ -4,14 +4,13 @@
     "name": "Openforis",
     "email": "openforis.arena@gmail.com"
   },
-  "version": "0.0.1",
+  "version": "0.0.19",
   "description": "Arena core module",
   "main": "index.js",
   "license": "MIT",
   "devDependencies": {
     "@babel/preset-typescript": "^7.13.0",
     "@types/jest": "^26.0.20",
-    "@types/uuid": "^8.3.0",
```

Comment:
Can types/uuid remain in devDependencies ?

Reply:
not sure, lets see